cfgov/jinja2/v1/owning-a-home/loan-estimate/index.html

```diff
@@ -23,9 +23,7 @@
       if ( document.body.parentElement.className.indexOf( 'no-js' ) === -1 ) {
         !function(){
           {# Include site-wide JavaScript. #}
-          var s = [
-            '{{ static('apps/owning-a-home/js/common.js') }}'
-          ];
+          var s = [ '/static/apps/owning-a-home/js/common.js' ];
```

Comment:
Why are we using the `static()` [elsewhere](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/jinja2/v1/_layouts/base.html#L68) but not here? Seems like both work, but it's the recommended way to use the helper ([see bullet 3](https://docs.djangoproject.com/en/2.0/howto/static-files/)).

Reply:
Generally, you are correct but I wasn't in favor of using it here because the file might resolve locally but fail during the build process.